### ISSUE TITLE
Bump react-native to 0.63.x

### ISSUE THREAD

**Issue by alexsegura** [coopcycle/coopcycle-app#723]
This would allow to support dark mode

Needs #722

**Comment by alexsegura** [comment]
There is a problem with React Native 0.63.x → the app can't be bundled for **release** on iOS. 

The following error occurs: 

```
File [...]/CoopCycle.app/main.jsbundle does not exist. 
This must be a bug with React Native, please report it here: https://github.com/facebook/react-native/issues
```

The actual problem is that, during the `Bundle React Native Code And Images` build phase, the script to generate the bundle triggers a syntax error: 

<img width="1340" alt="Capture d’écran 2020-09-05 à 15 37 42" src="https://user-images.githubusercontent.com/1162230/92306231-d28b6780-ef8d-11ea-8efe-7ff73635f5da.png">

There are some people reporting the same problem

- facebook/react-native#29351
- facebook/react-native#29477
- facebook/react-native#29636
- facebook/react-native#29818

**Comment by alexsegura** [comment]
The `master` branch has been reverted to use React Native `0.62`

The changes made for `0.63` have been backed up in the [bump-rn-63](https://github.com/coopcycle/coopcycle-app/tree/bump-rn-63) branch

**Comment by alexsegura** [comment]
Actually dark mode was fixed [here](9a01fb1ffc9dc5aade908318f7f7432773c5564b)

**Comment by alexsegura** [comment]
Version [0.63.3](https://github.com/react-native-community/releases/blob/master/CHANGELOG.md#v0633) is out, and it seems to fix the issue
